core/citrus-spring/src/main/java/com/consol/citrus/config/handler/CitrusTestcaseNamespaceHandler.java

```diff
@@ -28,7 +29,7 @@
  * Namespace handler registers bean definition parser
  * for Citrus testcase schema elements.
  *
- * @author Christoph Deppisch
+ * @author Christoph Deppisch, Thorsten Schlathoelter
  * @since 2007
  */
 public class CitrusTestcaseNamespaceHandler extends NamespaceHandlerSupport {
```

Comment:
I'd rename this class as `TestCase` is commonly written in camel-case. if @christophd thinks that this would not break anything?

```suggestion
public class CitrusTestCaseNamespaceHandler extends NamespaceHandlerSupport {
```

Reply:
I have my doubts about this, as this handler is reflectively created from spring.handlers.
Of course I can adjust it but I don't know if there is a use case where a user has explicitly configured a spring.handlers specifying this class. This would of course be a breaking change.

I leave it for @christophd to decide whether this cosmetic change is worth it.

Reply:
We should be fine to change the name as well as in spring.handlers in our the code base. The chance of someone referencing the old name in a custom spring.handlers is quite low. And if so the change is quite simple.

so ok with me

Reply:
@tschlat so renaming would be fine.. but it's ok if you feel uncomfortable about it.